lib/skywalking/util.lua

```diff
@@ -28,6 +28,21 @@ local split = function(str, delimiter)
     return t
 end
 
+
+local checkIgnoreSuffix = function(operationName, ignoreSuffix)
+    if ignoreSuffix ~= nil then
+        local ignore_suffix_table = split(ignoreSuffix, ",")
+        for _, suffix in ipairs(ignore_suffix_table) do
+            local l = string.find(operationName, suffix, string.len(operationName)  - string.len(suffix))
```

Comment:
@membphis @moonming What is the fastest way to check suffix? I am feeling `find` is not a good practice.

Reply:
https://github.com/apache/apisix/blob/master/apisix/core/string.lua#L58

this is the better way ^_^

Reply:
I think we have to copy its implementation to here, right?

Reply:
i got.

Reply:
@alonelaval If we have to copy it, please add comments on this method about the source. We(SkyWalking and APISIX) are owned by the same foundation, so, it is OK to copy without source code License change. But still, let's make the source clear.

Reply:
> @alonelaval If we have to copy it, please add comments on this method about the source. We(SkyWalking and APISIX) are owned by the same foundation, so, it is OK to copy without source code License change. But still, let's make the source clear.

ok.